lib/commands/smapi/smapi-command-handler.js

```diff
@@ -75,6 +75,24 @@ const _mapToParams = (optionsValues, flatParamsMap, commanderToApiCustomizationM
 
 const _sdkFunctionName = (swaggerApiOperationName) => `call${swaggerApiOperationName.charAt(0).toUpperCase() + swaggerApiOperationName.slice(1)}`;
 
+/**
+ * Parses response from smapi
+ * @param {Object} response object
+ */
+const parseSmapiResponse = (response) => {
+    let result = '';
+    const { body, headers } = response;
+    const contentType = headers.find((h) => h.key === 'content-type');
```

Comment:
Does it need to make sure the key is changed to all lower chase before comparison?

Reply:
https://nodejs.org/api/http.html.  Search for "Keys are lowercased. Values are not modified."

Reply:
https://nodejs.org/api/http.html#http_message_headers